README.md

```diff
@@ -175,6 +175,20 @@ Nim provides unique features for seamless and transparent interoperability with
 - [zero-functional](https://github.com/zero-functional/zero-functional) - A library providing (almost) zero-cost chaining for functional abstractions in Nim.
 
 
+### Pattern Matching
+
+- [npeg](https://github.com/zevv/npeg) - PEGs for Nim, another take.
+- [patty](https://github.com/andreaferretti/patty) - A pattern matching library for Nim.
+- [gara](https://github.com/alehander42/gara) - Macro-based pattern matching library.
+- [ast_pattern_match](https://github.com/krux02/ast-pattern-matching) - A library to do pattern matching on the AST.
+
+
+### Regex
```

Comment:
New heading needs to be added to the index.

Reply:
Done that too, also for the new section "path handling"